__tests__/app.test.js

```diff
@@ -33,16 +33,48 @@ describe("get /api", () => {
             .then(({ body }) => {
                 expect(typeof body.apiDoc === "object").toBeTruthy();
                 expect(typeof body.apiDoc).not.toBeNull();
-                expect(body.apiDoc).toHaveProperty("GET /api");
-                expect(body.apiDoc).toHaveProperty("GET /api/topics");
-                expect(body.apiDoc).toHaveProperty("GET /api/articles");
+                expect(body.apiDoc).hasOwnProperty("GET /api");
+                expect(body.apiDoc).hasOwnProperty("GET /api/topics");
+                expect(body.apiDoc).hasOwnProperty("GET /api/articles");
+                expect(body.apiDoc).hasOwnProperty("GET /api/articles/:article_id");
                 for (let i in body.apiDoc) {
-                    expect(body.apiDoc[i]).toHaveProperty("description");
+                    expect(body.apiDoc[i]).hasOwnProperty("description");
                 };
             });
     });
 });
 
+describe("get /api/articles/:article_id", () => {
+    test("returns an article object by id with appropriate content keys", () => {
+        const articleProps = ["author",
+            "title",
+            "article_id",
+            "body",
+            "topic",
+            "created_at",
+            "votes",
+            "article_img_url"]
+        return request(app)
+            .get("/api/articles/1")
+            .expect(200)
+            .then(({ body }) => {
+                articleProps.forEach(prop => {
+                    expect(body.article).hasOwnProperty(prop);
+                })
+            });
+    });
```

Comment:
This endpoint is for a specific article so we can and should be more specific with the expect statements. We know that the id should be 1, we also know that each of the other properties should exist on this object as well and we can also assert what these are. I would suggest taking a look at expect.objectContaining or .toMatchObject as well.